### PR TITLE
Updated changes for Ubuntu 18.04 LTS

### PR DESCRIPTION
**Pull request by rambabugovind**
Hey Connor,
I still have the Dell Inspiron 7559 laptop and installed Ubuntu 18.04 LTS on it. Your doc was really helpful and I thought I could update my experience with installing 18.04 LTS in the laptop. Thanks much.

### CODE REVIEW

File: dell-inspiron-7559-linux-guide/index.html [hunk 1]
```diff
@@ -1,258 +1,440 @@
...
+                <h1>
+                        <a name="top-title">Dell Inspiron 7559 Ubuntu Linux Guide</a>
+                </h1>
+                <p>Last updated
+                        <b>10 July 2018</b>
+                </p>
+                <p>This guide will show you how to install and configure an
+                        <strong>Ubuntu Linux 16.04/18.04 LTS</strong> distribution (Ubuntu, Lubuntu, Xubuntu, Kubuntu, Ubuntu Gnome, Ubuntu
+                        MATE) on the Dell Inspiron 7559 laptop.</p>
+                <p class="note note-warn">Donot try to upgrade from
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: "Donot" -> "Do not"

File: dell-inspiron-7559-linux-guide/index.html [hunk 2]
```diff
@@ -1,258 +1,440 @@
...
+                <p>Last updated
+                        <b>10 July 2018</b>
+                </p>
+                <p>This guide will show you how to install and configure an
+                        <strong>Ubuntu Linux 16.04/18.04 LTS</strong> distribution (Ubuntu, Lubuntu, Xubuntu, Kubuntu, Ubuntu Gnome, Ubuntu
+                        MATE) on the Dell Inspiron 7559 laptop.</p>
+                <p class="note note-warn">Donot try to upgrade from 
+                        <b>Ubuntu 16.04 LTS</b> to <b>18.04 LTS now</b>. This might crash your OS (Crashed mine! Had to do a fresh install of
+                        Ubuntu 18.04 LTS.). Upgrade from 16.04 LTS to 18.04 LTS will be enabled 
+                <a href="https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes">late july</a>.
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: I'd suggest linking directly to the relevant section: [https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Upgrading_from_Ubuntu_16.04_LTS](https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes#Upgrading_from_Ubuntu_16.04_LTS)

File: dell-inspiron-7559-linux-guide/index.html [hunk 3]
```diff
@@ -1,258 +1,440 @@
...
-                                <p class="note note-info">This <span class="in-mono">wget</span> command will download the correct headers and image files for the 64bit 4.8.5 kernel into the working directory.</p>
-                                <p class="note note-warn"><strong>WARNING</strong> Be absolutely certain there are no other .deb files in the current directory before this next step.</p>
-                                <li>Install the new kernel.</li>
-                                <code>sudo dpkg -i *.deb</code>
-                                <li>Reboot.</li>
-                        </ol>
+                <h2>
+                        <a name="toc-updatekernel">Update to Linux kernel 4.8.5 for better GTX 900 & Optimus support</a>
+                </h2>
+                <p class="note note-18"><b>Didnot</b> test this section with <b>18.04 LTS</b> installation.</p>
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: "Didnot" -> "Did not"

File: dell-inspiron-7559-linux-guide/index.html [hunk 4]
```diff
@@ -1,258 +1,440 @@
...
-                                        OpenGL shading language version string: 4.50 NVIDIA <br />
-                                </code>
-                                <li>Verify that the discrete card is now off since we aren't actively running a program with <span class="in-mono">optirun</span>.</li>
-                                <code>cat /proc/acpi/bbswitch</code>
-                                <p>We should get an output like:</p>
-                                <code>
+                <h2>
+                        <a name="toc-bumblebee">Installing Bumblebee</a>
+                </h2>
+                <p class="note note-18"><b>Didnot</b> test this section with <b>18.04 LTS</b> installation.</p>
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: "Didnot" -> "Did not"

File: dell-inspiron-7559-linux-guide/index.html [hunk 5]
```diff
@@ -1,258 +1,440 @@
...
-                                <code>tar -xf name_of_downloaded_file_here</code>
-                                <p>You'll need to perform that command for both archives.</p>
-                                <li>Go into the extracted folders one at a time and install the firmwares.</li>
-                                <code>sudo bash ./install.sh --install</code>
-                                <li>Reboot</li>
-                        </ol>
+                <h2>
+                        <a name="toc-inteldriver">(Optional) Installing the Intel Graphics Driver and Firmware</a>
+                </h2>
+                <p class="note note-18"><b>Didnot</b> test this section with <b>18.04 LTS</b> installation.</p>
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: "Didnot" -> "Did not"

File: dell-inspiron-7559-linux-guide/index.html [hunk 6]
```diff
@@ -1,258 +1,440 @@
...
-                                <li>Right click on the game in your Steam library and click Properties.</li>
-                                <li>Click Set Launch Options.</li>
-                                <li>Enter <span class="in-mono">primusrun %command%</span>.</li>
-                                <p class="note note-info">If you already have launch options in this box, make sure that you put <span class="in-mono">primusrun</span> at the very beginning and put <span class="in-mono">%command%</span> at the very end and everything should work fine.</p>
-                                <li>Play your game normally by launching it through the Steam client!</li>
-                        </ol>
+                <h2>
+                        <a name="toc-gpuswitching">Running games and applications with the proper GPU</a>
+                </h2>
+                <p class="note note-18"><b>Didnot</b> test this section with <b>18.04 LTS</b> installation.</p>
```
Comment (connorkuehl) [inline dell-inspiron-7559-linux-guide/index.html]: "Didnot" -> "Did not"

**Review by connorkuehl — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute to this!

I like the changes you made, but there are just a couple of typos I'd like fixed before I merge this.

Also, the blue boxes that say "Follow the below steps for 18.04 LTS installation as well." seem redundant due to your disclaimer at the beginning that says "Text within blue boxes indicates instructions intended only for Ubuntu Linux 18.04 LTS. The sections that were tested with Ubuntu Linux 18.04 LTS will be explicitly stated. "

**Comment by rambabugovind** [comment]
Thanks a lot for getting back and thanks for letting me know of the changes. I’ll make the relevant changes and open a new PR.

**Comment by connorkuehl** [comment]
You can just push to your branch and this PR will be updated automatically.

**Comment by rambabugovind** [comment]
Hey Connor, I have fixed the mentioned typos and the link. Let me know if there are any other changes.

**Review by connorkuehl — APPROVED** [review]

**Comment by connorkuehl** [comment]
Thank you!

**Pull request merged**